crossdc-consumer/build.gradle

```diff
@@ -60,8 +60,8 @@ dependencies {
         exclude group: "org.apache.logging.log4j", module: "*"
         exclude group: "org.slf4j", module: "*"
     }
-    testImplementation 'org.apache.kafka:kafka_2.13:2.8.1'
```

Comment:
this change seems unrelated

Reply:
It's to fix being able run properly in the ide, which is related to debugging.